WordPress/Classes/ViewRelated/Site Creation/Final Assembly/SiteAssemblyWizardContent.swift

```diff
@@ -319,6 +319,11 @@ extension SiteAssemblyWizardContent: NUXButtonViewControllerDelegate {
             return
         }
 
+        // Disable the prompt for WordPress when the blog has no domains.
+        guard AppConfiguration.isJetpack || blog.hasDomains else {
```

Comment:
Why do we use `blog.hasDomains`? What does domains have to do with it?
My understanding is we are disabling this prompt for WordPress because the dashboard is disabled. To stay on the safe side, we can use `BlogDetailsViewControllerisDashboardEnabled()`. What do you think?

Reply:
I duplicated that function as I didn't want to cause regression on call-sites.